extensions/ql-vscode/src/variant-analysis/variant-analysis-manager.ts

```diff
@@ -342,6 +327,32 @@ export class VariantAnalysisManager
       );
     }
 
+    // It's not possible to interpret a BQRS file to SARIF without an id property.
+    if (
+      queryMetadata?.kind &&
+      ["problem", "path-problem"].includes(queryMetadata.kind) &&
```

Comment:
Should we also include `alert` and `path-alert` in this list?

Reply:
Yes, good point. I've changed this to use a function instead and also used this function in other places where I could find `path-problem`.